packages/editor/src/components/provider/index.js

```diff
@@ -120,7 +119,6 @@ function EditorProvider( {
 					>
 						{ children }
 						<ReusableBlocksMenuItems />
-						<ConvertToGroupButtons />
```

Comment:
Hmm, so it was placed next to reusable blocks in the menu already. Now, I'm curious why it isn't shared between edit post, edit site, and edit widgets?

Reply:
Actually reusable blocks are used in editor, widgets and there is a similar but custom implementation in site-editor that 'maps' reusable blocks -> template parts. So for reusables might make sense not be in all editors.

I'll investigate today for a solution of this, but I'm curious why `convert to group` wasn't in `block-editor` from the beginning.

Reply:
it wasn't in block-editor because defining a default  group block was added after the group block was added.

Reply:
Thanks for explaining 👍